ui/scss/init/_gui.scss

```diff
@@ -2061,28 +2061,6 @@ input:not(input:-webkit-autofill)::-webkit-contacts-auto-fill-button {
     margin-top: 10px;
   }
 
-  .banner_logo {
-    display: block;
-    float: left;
-    margin-top: 20px;
-    margin-left: 80px;
-    margin-right: 30px;
-    width: 150px;
-    height: 150px;
-    background-image: url('/public/img/cookie.svg');
```

Comment:
In this case, the image reference couldn't be found by webpack. I start poking through to see if it wasn't getting copied, but then I realized that I could find any references to `.banner_logo` in the codebase. Need an extra eye here to confirm that I can remove this, the cache control reference and the image itself.